aperturedb/Utils.py

```diff
@@ -663,6 +663,42 @@ def count_descriptors_in_set(self, set_name):
 
         return total
 
+    def remove_all_indexes(self):
+        """Remove all indexes from the database.
+
+        This may improve the performance of remove_all_objects.
+        It may improve or degrade the performance of other operations.
+
+        Note that this only removes populated indexes.
+        """
+        def find_indexes(schema):
```

Comment:
Sorry this was an oversight. There's already a similar implementation 
https://github.com/aperture-data/aperturedb-python/blob/70fb10399128a6c525aa2a9990cf1f689861eaf4/aperturedb/ParallelLoader.py#L22

and remove index itself exists in utils:
https://github.com/aperture-data/aperturedb-python/blob/70fb10399128a6c525aa2a9990cf1f689861eaf4/aperturedb/Utils.py#L240

Can this all be reconciled? @bovlb

Reply:
Utils might be a good place to put this in.

Reply:
I'll consolidate `get_existing_indices` with `find_indexes`.
I didn't want to use `_remove_index` because it generates a query per index.